SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Supported Projects
+
+We will provide fixes for security issues affecting:
+
+- [Shields.io](https://shields.io)
+- Users running their own self-hosted Shields instance
```

Comment:
Could we maybe use some kind of catch all that incorporate Shields-specific code and official/distributed artifacts (our docker image), or at least some qualifying text? I feel that second item in particular could be misinterpreted as too broad of a scope (i.e. issues within Node versions, dependency tree, anything on the OS side, etc.)

Reply:
Also should we include the [svg proxy](https://github.com/badges/svg-to-image-proxy) here? I know that code lives in a separate repo but we point folks to the main repo here for issues, etc.

Reply:
I've added the svg proxy and made a tweak to the wording about self-hosting. More generally, 
do you have any suggestions for better wording here?

We've already got a sentence that says "Report security bugs in third-party modules to the person or team maintaining the module"

Also bear in mind the purpose of a SECURITY.md is

> Help your community understand how to securely report security vulnerabilities for your project

Its not a contract. If someone reports something to us that in fundamentally a bug in docker/node/whatever it doesn't mean we've committed to fixing that because someone has decided our markdown file is poorly worded.

Reply:
In fact given that purpose, maybe we should be even looser than "we will provide fixes for security issues affecting..."

maybe its more like "you can report security issues affecting..." ??

Reply:
Loosening suggestion: "we will provide" -> "we will aim at providing"?

Reply:
> Also bear in mind the purpose of a SECURITY.md is

>>    Help your community understand how to securely report security vulnerabilities for your project

> Its not a contract. If someone reports something to us that in fundamentally a bug in docker/node/whatever it doesn't mean we've committed to fixing that because someone has decided our markdown file is poorly worded.

Fair enough, but the opening sentence made the doc read like it was covering what we will fix, not how users can report vulnerabilities to us.

> maybe its more like "you can report security issues affecting..." ??

That's more aligned with what I had in mind (I was going to suggest text but couldn't quite articulate the words).

Perhaps along the lines of your suggestion:

```
For security issues affecting, or, this security reporting policy covers..
 - Shields.io
 - ....

etc
```

Reply:
new version pushed